reconcile/terraform_tgw_attachments.py

```diff
@@ -162,13 +218,52 @@ def run(
     if print_to_file:
         sys.exit()
 
-    aws_api = AWSApi(1, accounts, settings=settings, init_users=False)
+    return working_dirs
+
+
+@defer
+def run(
+    dry_run: bool,
+    print_to_file: Optional[str] = None,
+    enable_deletion: bool = False,
+    thread_pool_size: int = 10,
+    defer: Optional[Callable] = None,
+):
+    settings = queries.get_secret_reader_settings()
+    clusters = queries.get_clusters_with_peering_settings()
```

Comment:
Can we convert this to also use a typed query, e.g., https://github.com/app-sre/qontract-reconcile/blob/master/reconcile/typed_queries/clusters.py or create a new one that better fits this purpose? For deps that require a nested untyped mapping, you can then use `clusters.dict(by_alias=True)` to stay backwards compatible - we do it in a couple of places. However, it will make refactoring the deps easier later on if the integrations already support the typed queries.

Reply:
(basically, we try to reach a point where we can remove `queries.py` - just ~10 more years to go :) )

Reply:
One more thing: We have a helper class to create generated query classes for tests, [gql_class_factory](https://github.com/app-sre/qontract-reconcile/blob/master/reconcile/test/conftest.py#L172) - it is very handy to reduce boilerplate, as some generated classes are veeery big and nested. So this might be handy if you need to create some cluster classes in tests.